src/plugins/inspector/public/views/requests/components/requests_view.tsx

```diff
@@ -153,6 +153,21 @@ export class RequestsViewComponent extends Component<InspectorViewProps, Request
           </EuiText>
         )}
 
+        {this.state.request && this.state.request.searchSessionId && (
```

Comment:
Is this only for the functional tests, or do you think there is value in showing this parameter to the user?

Reply:
I think it would also be useful for debugging and support purposes